.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!-- Hello! Thanks for submitting a PR. To help make things go a bit more
+     smoothly we would appreciate that you go through this template.
+-->
+
+### Description
+
+<!-- Good things to put here include: reasoning for the change (please link
+     any relevant issues!), any noteworthy (or hacky) choices to be aware of,
+     or what the problem resolved here looked like ... we won't mind a ranty
+     story :)
+-->
```

Comment:
I would add:
 if it's a new functionality, the algorithm you chose to implement it

Reply:
I think that's a little too specific to include here. How about a checkbox to indicate that, if documentation is being added or modified, you've successfully built to docs locally? We could include a link to instructions about how to build the docs locally.